apps/openchallenges/edam-etl/pyproject.toml

```diff
@@ -11,12 +11,9 @@ python = "3.12.0"
 pandas = "2.2.1"
 requests = "2.31.0"
 regex = "2023.12.25"
-
-[tool.poetry.group.dev.dependencies]
-
-[tool.poetry.group.prod.dependencies]
-
-[tool.poetry.group.test.dependencies]
+mysql-connector = "^2.2.9"
```

Comment:
The version should be pinned. You can't simplify change this value, instead you should reinstall the package with `poetry` while specifying the version: `poetry add mysql-connector@2.2.9`.

Same comments for the packages below.

Reply:
We discussed the problems with using poetry to install this package back in May. It did not work. I will try again and hopefully there has been a poetry update to improve functionality when working with Maria DB and mysql.